jdk-http-client/src/main/scala/org/http4s/client/jdkhttpclient/JdkHttpClient.scala

```diff
@@ -0,0 +1,109 @@
+package org.http4s.client.jdkhttpclient
+
+import java.net.URI
+import java.net.http.HttpRequest.BodyPublishers
+import java.net.http.HttpResponse.BodyHandlers
+import java.net.http.{HttpClient, HttpRequest, HttpResponse}
+import java.nio.ByteBuffer
+import java.util
+import java.util.concurrent.{CancellationException, CompletableFuture, CompletionException, Flow}
+import java.util.function.BiFunction
+
+import cats.effect._
+import cats.implicits._
+import fs2.interop.reactivestreams._
+import fs2.{Chunk, Stream}
+import org.http4s.client.Client
+import org.http4s.util.CaseInsensitiveString
+import org.http4s.{Header, Headers, HttpVersion, Request, Response, Status}
+import org.reactivestreams.FlowAdapters
+
+import scala.collection.JavaConverters._
+
+object JdkHttpClient {
+
+  def apply[F[_]](builder: HttpClient.Builder = HttpClient.newBuilder)(
+      implicit F: ConcurrentEffect[F]): F[Client[F]] =
+    F.delay(builder.build).map { jdkHttpClient =>
```

Comment:
Building the http client is an effect (default Executor creation, SSLContext).

Reply:
The `BlazeClientBuilder` takes an `ExecutionContext` as an argument instead of returning an `Effect`.  We might consider that pattern here.